catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Association.js

```diff
@@ -142,15 +143,17 @@ define([
         },
         removeSegment: function (segment) {
```

Comment:
Thanks for the insight. I haven't used backbone-relational before. Right now the whole ui model structure is based on backbone-associations (more than just Association.js). Changing to backbone-relational seem more like a module refactor so I don't think I will change it up here with this update. If we change it up we should probably re-examine the whole module and refactor it to use backbone-relational in several places.